src/Biscotti/Session/Store/Memory.purs

```diff
@@ -65,9 +66,8 @@ get store cookie = do
   pure do
     key <- getKey cookie
     val <- note "session not found" $ Map.lookup key map
-    json <- jsonParser val
-
-    decodeJson json
+    
+    lmap printJsonDecodeError $ decodeJson =<< parseJson val
```

Comment:
Since `decodeJson` now uses `JsonDecodeError`, here I'm using `printJsonDecodeError` to retrieve a `String`. However, it's also possible for this library API to change so that the `Getter` class (and other classes) also use the `JsonDecodeError` type. That said, in the interest of minimal impact on downstream libraries I opted to just recover the string.